docs/section19/19-13.md

```diff
@@ -0,0 +1,82 @@
+            - Get list of zones served by the hvac system: `zones_list_hvac_sys_b = dict_of_zones_and_terminal_units_served_by_hvac_sys_b[hvac.id]["ZONE_LIST"]`  
+            - Get list of terminal units served by the hvac system: `terminal_list_hvac_sys_b = dict_of_zones_and_terminal_units_served_by_hvac_sys_b[hvac.id]["ZONE_LIST"]["Terminal_Unit_List"]`  
+
+            Below determines the zone setpoint to use in the delta T calculation for systems with multiple zone thermostat setpoints, use the design set point that will result in the lowest supply air cooling set point or highest supply air heating set point: 
+            - Reset the design_thermostat_cooling_setpoint variable from the previous HVAC system (not sure how to do this with a numeric variable): `design_thermostat_cooling_setpoint = ""`  
+            - Reset the design_thermostat_heating_setpoint variable from the previous HVAC system (not sure how to do this with a numeric variable): `design_thermostat_heating_setpoint = ""`  
+            - Reset counter variable: `x=0`
+            - Cycle through each zone associated with the HVAC system to get the lowest design_thermostat_cooling_setpoint and the highest design_thermostat_heating_setpoint: `for zone in zones_list_hvac_sys_b:`  
+                - Get zone_design_thermostat_cooling_setpoint: `zone_design_thermostat_cooling_setpoint = zone.design_thermostat_cooling_setpoint`
+                - Get zone_design_thermostat_heating_setpoint: `zone_design_thermostat_heating_setpoint = zone.design_thermostat_heating_setpoint`
+                - Check if x = 0, this means this is the first zone associated with the HVAC system: `if x == 0:`  
+                    - Set the design_thermostat_cooling_setpoint equal to this zone's value: `design_thermostat_cooling_setpoint = zone_design_thermostat_cooling_setpoint`
+                    - Set the design_thermostat_heating_setpoint equal to this zone's value: `design_thermostat_heating_setpoint = zone_design_thermostat_heating_setpoint`     
+                - Else, means that this is not the first zone so need to compare this zone's values to the current values: `Else:`  
+                    - Check if the zone_design_thermostat_cooling_setpoint is less than the design_thermostat_cooling_setpoint: `if zone_design_thermostat_cooling_setpoint < design_thermostat_cooling_setpoint: design_thermostat_cooling_setpoint =  zone_design_thermostat_cooling_setpoint`                 
+                    - Check if the zone_design_thermostat_heating_setpoint is greater than the design_thermostat_heating_setpoint: `if zone_design_thermostat_heating_setpoint > design_thermostat_heating_setpoint: design_thermostat_heating_setpoint =  zone_design_thermostat_heating_setpoint`  
```

Comment:
The above logic can be simplified to:
```python
design_thermostat_heating_setpoint = -infinity
design_thermostat_cooling_setpoint = +infinity
for zone in zone_list_hvac_sys_b:
    if zone.zone_design_thermostat_cooling_setpoint < design_thermostat_cooling_setpoint:
        design_thermostat_cooling_setpoint = zone.zone_design_thermostat_cooling
    if zone.zone_design_thermostat_heating_setpoint > design_thermostat_heating_setpoint:
        design_thermostat_heating_setpoint = zone.zone_design_thermostat_heating
```
This will guarantee to get the lowest cooling setpoint and highest heating setpoint in the list of zones.

Reply:
Addressed